app/build.gradle

```diff
@@ -52,7 +52,7 @@ dependencies {
     compile 'com.google.android.gms:play-services-gcm:11.0.1'
 
     compile group: 'com.zopim.android', name: 'sdk', version: '1.3.4.1'
-    compile group: 'com.zendesk', name: 'sdk', version: '1.9.2.1'
+    compile 'com.zendesk:sdk:1.10.0.1'
```

Comment:
Can you change this back to the separate group, name, and version syntax as above?